README.md

```diff
@@ -196,9 +201,9 @@ with your preferred flavour.
 -   `kubuntu` (Kubuntu)
 -   `lubuntu` (Lubuntu)
 -   `ubuntu-budgie` (Ubuntu Budgie)
--   `ubuntu-kylin` (Ubuntu Kylin)
+-   `ubuntukylin` (Ubuntu Kylin)
```

Comment:
@flexiondotorg This generated Readme has picked up the changes from merging #387 .